ruby-push-notifications.gemspec

```diff
@@ -1,11 +1,7 @@
 # coding: utf-8
-lib = File.expand_path('../lib', __FILE__)
-$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
-require 'ruby-push-notifications/version'
-
 Gem::Specification.new do |spec|
   spec.name          = "ruby-push-notifications"
-  spec.version       = RubyPushNotifications::VERSION
```

Comment:
To avoid having to require the gem's files in this file. It is a recommendation I took from ruby gems.org